docs/NewFeatures/CRR_statistics.rst

```diff
@@ -0,0 +1,10 @@
+S3 Cross-Region Replication (CRR) Statistics
+--------------------------------------------
+
+This feature enables replication status tracking. The replication system
+exposes metrics through a REST API to monitor the replication of
+pending, processing, and completed objects. It returns the number of
+errors that occurred during replication, the current throughput—in

```

Comment:
Maybe "failures" is a more accurate word choice compared with "errors".

Reply:
Done.